ngraph/frontend/paddlepaddle/src/frontend.cpp

```diff
@@ -266,6 +268,12 @@ InputModel::Ptr FrontEndPDPD::load_impl(const std::vector<std::shared_ptr<Varian
     PDPD_THROW("Model can be loaded either from 1 or 2 files/streams");
 }
 
+void FrontEndPDPD::normalize(std::shared_ptr<ngraph::Function> function) const {
+    ngraph::pass::Manager m;
+    m.register_pass<ngraph::pass::MOCTransformations>(false);
```

Comment:
should not they be moved to ie transformations then?

Reply:
yes, indeed

Reply:
done